Clarification for Testing Environment Variables (#38359)
vercel/next.js: docs/basic-features/environment-variables.md

## Test Environment Variables

> Apart from `development` and `production` environments, there is a 3rd option available: `test`. In the same way you can set defaults for development or production environments, you can do the same with a `.env.test` file for the `testing` environment (though this one is not as common as the previous two). Despite this, note that NextJS will not load variables from `development` or `production` in the testing environment (meaning values from `.env.development` and `env.production` will be ignored).

Reviewer: ```suggestion
Apart from `development` and `production` environments, there is a 3rd option available: `test`. In the same way you can set defaults for development or production environments, you can do the same with a `.env.test` file for the `testing` environment (though this one is not as common as the previous two). Next.js will not load environment variables from `.env.development` or `.env.production` in the `testing` environment.
```
